tests/parsers/test_pw.py

```diff
@@ -210,6 +210,27 @@ def test_pw_failed_computing_cholesky(fixture_localhost, generate_calc_job_node,
     assert calcfunction.exit_status == node.process_class.exit_codes.ERROR_COMPUTING_CHOLESKY.status
 
 
+def test_pw_failed_computing_cholesky_stdout(
```

Comment:
I see that you only added this for the failed cholesky test, but not for the other tests where you moved the error from the stdout to the crash file. What is your reasoning here? Couldn't we have just kept the old tests, parsing from stdout, and _add_ a test where the error is (only) in the crash file to test cases for QE 7.2?

Reply:
I thought, as this will become the standard, to put all the errors in the CRASH, which seemed cleaner to me - no particular reasoning though. 

I replicated only one error as I would assume that if it works for one, it will for all the others - thus no need to have more tests. Or do you think we should have duplicate tests both for CRASH and stdout?

Reply:
Ok, because I also wanted to add more errors in a next PR, which are belonging to diagonalization errors (for `ppcg` and `paro` mainly).

So, should we leave it like this for now?

Reply:
I didn't notice that the `vcrelax` errors had different folder names. While moving their error in CRASH, the tests are not passing. So, I think at this point it is convinient to have duplicated tests?

Reply:
In the last commit, there are tests for both ``CRASH`` and ``stdout`` cases